### ISSUE TITLE
Apollo 10 missing PAD and late uplinks

### ISSUE THREAD

**Issue by RL77** [dseagrav/NASSP#418]
Hello

I want to report a possible bug in the apollo 10 mission near the GET for the LEM undoking, after  this Pad appears:

https://puu.sh/C9Pow/66e401d50a.png?fbclid=IwAR1ji9awKp76kiVZbymuRndq2xtGpxKnUT6wM1-NrjqIGIXLV9k2C8qeVuI

A little bit  later a message appears informing of another Pad (saying "you got Pad" or similar!). But this Pad does not appear! I think this Pad should contain information about the time of undocking and others values.
After that, and due to the fact that the LEM and CSM  uplinks just before undock, appear later than expected in the checklist, I cannot get everything ready by the time of the undocking, scheduled in the checklist and flight plan of Apollo 10 at Get 98h10. I can only be prepared 20 minutes later on GET+- 98h30.

https://puu.sh/C9RXe/4243d15041.png?fbclid=IwAR0gLc0_AYeUAFBRpNYzTjfquja-xJeUqaxvtV9ztXLR-HYSxXNQPzhV8zI

Not counting on the possible Pad that is missing, it is possible to start undocking with 20 minutes of delay?
I did not do the undoking , because I have not had time yet ,and I am in doubt if I still can do the undocking  because of the delay. But even so I accelerated the time a little more and appeared the pad for the DOI maneuver  in the LEM. With the delay, will I be able to  perform the rest of the procedures to do DOI on time?

https://puu.sh/C9Svf/ef054adf7e.png?fbclid=IwAR2NSPMnJ-GD7s5vQFYKa1uWVUYj5oAlfEkcrX-OwyKjzFKxKu2XaCuQv0s

In short, I think there is a missing Pad with information between these two Pads that I show in the prints, and the uplinks come too late to be ready for undoking in GET 98h10, unless there is no problem of making  the undocking a little late in GET+- 98h30. Can you give me some help here?

Thanks you very much

**Comment by indy91** [comment]
Undocking late is very much possible and even desired in your case. You arrived late in lunar orbit and all times should be relative to PDI time on a lunar mission. In the case of Apollo 10 this will be the time of periapsis after DOI. You can compare the LOS/AOS times from the flight plan to see how late or early you are compared to it. Or to quote the Apollo 10 transcript:

> 070:19:20 Lousma: That is affirmative. Also, we would like you to know that LOI is now about 11 minutes later than our preflight planning, because we didn't burn midcourse correction 1 and made our translunar trajectory adjustment at midcourse 2 instead. So this puts us 11 minutes behind on LOI and 11 minutes behind throughout all of our lunar orbital activities.

Same applies to you, although you will be (as currently expected in NASSP) even more than 11 minutes late with all lunar orbit activites. Or else you will have some 20 minutes idle time before DOI somewhere.

I've quickly looked at the code and there definitely is something missing there with the PAD. I'll check what exactly and will fix it soon.

**Comment by RL77** [comment]
You welcome, I'm glad I can help !! :) , and I will continue to help, if I find more I will report! I am going to wait for you to do the correction of the pad and then I  will continue the mission. And thanks again for your great explanations!

**Comment by indy91** [comment]
It's been fixed in the newest build.

**Comment by RL77** [comment]
Great! :D Thanks indy91!
